README.md

```diff
@@ -95,6 +95,42 @@ You can use a service like [Polyfill.io](https://polyfill.io) to download only t
 <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=Element.prototype.classList"></script>
 ```
 
+### Shadow DOM
+
+It could be very expensive to apply this polyfill automatically to every shadow
+root that is created in a given document, so the polyfill ignores shadow roots
+by default. If you are using Shadow DOM in a component, it is possible to apply
+this polyfill imperatively to the component's shadow root:
+
+```javascript
+// Check for the polyfill:
+if (self.applyFocusVisiblePolyfill != null) {
```

Comment:
Is `self` equivalent to `window`? Not sure if the snippet makes it clear what context this is running in.

Reply:
Yeah, `self` is equivalent to `window` in the document context. I can change this to `window`.